backend/src/models/users.go

```diff
@@ -157,4 +157,33 @@ type ResidentProgramClassInfo struct {
 	ProgramID            uint                    `json:"program_id"`
 	PresentAttendance    int                     `json:"present_attendance"`
 	AbsentAttendance     int                     `json:"absent_attendance"`
+	CreditTypes          string                  `json:"credit_types"`
+}
+
+type ResidentProgramClassHistory struct {
+	ProgramName       string      `json:"program_name"`
+	ClassName         string      `json:"class_name"`
+	Status            ClassStatus `json:"status"`
+	DateStatusChanged string      `json:"date_status_changed"`
+	CreditTypes       string      `json:"credit_types"`
+}
```

Comment:
@corypride We shouldn't need this if we use the one handler in `user_handler.go` for `/programs`